### PR TITLE
move text to i18n

### LINKED ISSUE #71: The Chinese translation text seems incomplete
## What's going wrong?

The Chinese version of the site hasn't been completed yet. There're lots of English text when user changes his language to Chinese.

## How to reproduce?

No need for this issue.

## Expected behaviour

There should not be a dozen of English text in Chinese version.

## Which version?

Branch `refactor`.

### PR DESCRIPTION
**Pull request by codeAtNow**
move text resources to i18n

### CODE REVIEW

**Comment by SantanaSP** [comment]
So what does this PR do and which issue is related?

**Comment by codeAtNow** [comment]
> So what does this PR do and which issue is related?

There is no issues related,just refactor some code

**Comment by SantanaSP** [comment]
Um, it seems be related to issue #71 . I'll add a link if you don't mind.

**Comment by SantanaSP** [comment]
And give me some minutes to review the changes.

File: packages/site/src/routes/TheIndex.vue [hunk 1]
```diff
@@ -43,7 +43,7 @@ function calculateStyle() {
     left: 281px;
     width: 310px;
     height: 310px;
-    background-image: url("home/block_1.svg");
+    background-image: url("/assets/home/block_1.svg");
```
Comment (SantanaSP) [inline packages/site/src/routes/TheIndex.vue]: Don't add modifications don't belong to current issue except those are related to formatting. If you'd like to deal with the background image in index page, please open a new PR.

**Review by SantanaSP — CHANGES_REQUESTED** [review]

File: packages/site/src/routes/TheIndex.vue (hunk 1 above)
Comment (codeAtNow) [reply packages/site/src/routes/TheIndex.vue]: OK,I didn't see the #71 PR

**Comment by codeAtNow** [comment]
I refactor some code for issue #71,But not finish

File: packages/site/src/routes/TheIndex.vue (hunk 1 above)
Comment (SantanaSP) [reply packages/site/src/routes/TheIndex.vue]: Pls dont resolve this conversation before we actually resolve the problem.

**Comment by SantanaSP** [comment]
> I refactor some code for issue #71,But not finish

Stash uncommitted code, and switch your branch

**Comment by SantanaSP** [comment]
But next time you should use two branches

File: packages/site/src/routes/TheIndex.vue (hunk 1 above)
Comment (codeAtNow) [reply packages/site/src/routes/TheIndex.vue]: I understand,and I push another commit to remove the background code

**Review by SantanaSP — APPROVED** [review]

**Pull request merged**